worker.go

```diff
@@ -125,6 +133,23 @@ func recvCb(buf unsafe.Pointer, buflen C.int, index workerTableIndex) C.buf {
 	}
 }
 
+//export resolveModule
+func resolveModule(moduleSpecifier *C.char, referrerSpecifier *C.char, resolverToken int) C.int {
```

Comment:
Capitalize: ResolveModule

resolverToken shadows the global variable - which will be fixed after the rename I suggested above.

(Running "go vet" should catch issues like this.)